assignment08/index.php

```diff
@@ -0,0 +1,29 @@
+
+<?php
+    include "includes/header.php";
+    $my_weight= '125 lb';
+    $_age = 25;
+    $first_name='Donjeta';
+    $last_name='Emerllahu';
+    $poundsToKg= 'pounds *(1.2.2)';
```

Comment:
I'm not sure why you have this here. This is just a string of the formula for conversion.